### ISSUE TITLE
Loss value is going to be negative.

### ISSUE THREAD

**Issue by DataX-JieHao** [czifan/DeepSurv.pytorch#5]
https://github.com/czifan/DeepSurv.pytorch/blob/f572a8a7d3ce5ad10609bd273ce3accaf7ea4b66/networks.py#L76

Would it be log_loss = torch.sum(log_loss, dim=0)?

**Comment by ml-and-ml** [comment]
loss is negative for me, though i'm unsure why the training will works. curious about above question^

**Comment by czifan** [comment]
> https://github.com/czifan/DeepSurv.pytorch/blob/f572a8a7d3ce5ad10609bd273ce3accaf7ea4b66/networks.py#L76
> 
> Would it be log_loss = torch.sum(log_loss, dim=0)?

Yes, you're right. I made a mistake here. Thanks for pointing out!!!

**Comment by czifan** [comment]
> loss is negative for me, though i'm unsure why the training will works. curious about above question^

Correcting Loss according to the above will not cause this problem.
